backend/userAPI/src/task/dto/create-task.dto.ts

```diff
@@ -6,10 +6,13 @@ export enum TaskType {
 }
 
 export enum TaskAction {
-	GITHUB_PR_MERGE = 'GITHUB_PR_MERGE',
-	GITHUB_ISSUE_CLOSE = 'GITHUB_ISSUE_CLOSE',
+	GITHUB_PR_OPEN = 'GITHUB_PR_OPEN',
+	GITHUB_ISSUE_OPEN = 'GITHUB_ISSUE_OPEN',
 	TIMER_DATE = 'TIMER_DATE',
 	TIMER_INTERVAL = 'TIMER_INTERVAL',
+	GOOGLE_CALENDAR_NEW_EVENT = 'GOOGLE_CALENDAR_NEW_EVENT',
+	GOOGLE_NEW_DOC = 'GOOGLE_NEW_DOC',
+	GOOGLE_NEW_SHEET = 'GOOGLE_NEW_SHEET'
```

Comment:
Same as above.

Reply:
Fixed in https://github.com/Cleopha/ifttt-like/pull/42/commits/5bd497b96dbec7651fb3c60fdd9d87ffbf3a89de